backend/src/main/java/com/example/demo/LoginRegister/UserDTO.java

```diff
@@ -0,0 +1,4 @@
+package com.example.demo.LoginRegister;
```

Comment:
Why did you copy this class? It seems like it's never used.

Reply:
deleted it